pkg/kubelet/kubelet_test.go

```diff
@@ -2598,7 +2598,7 @@ func TestHandlePodResourcesResize(t *testing.T) {
 	for _, tt := range tests {
 		tt.pod.Spec.Containers[0].Resources.Requests = tt.newRequests
 		tt.pod.Status.ContainerStatuses[0].AllocatedResources = v1.ResourceList{v1.ResourceCPU: cpu1000m, v1.ResourceMemory: mem1000M}
-		kubelet.handlePodResourcesResize(tt.pod)
+		kubelet.handlePodResourcesResize(tt.pod, &tt.pod.Status)
```

Comment:
em. I know we optimize performance in this PR and may not need to add additional test. However, we do change logics in `determinePodResizeStatus`. Can we add some tests to cover those changes?

Reply:
done